guestbook/commentapp/tests.py

```diff
@@ -0,0 +1,101 @@
+from django.test import TestCase
+from datetime import timedelta
```

Comment:
More of a convention / suggestion / recommendation than anything wrong: use `import datetime` and `datetime.timedelta`. If you use your way in e.g. `from datetime import datetime` it isn't clear anymore if `datetime` is the upmost package or a module. Thus it's often said to only use `import datetime`.